docs/developers/protocol/hermez-protocol/circuits/circuits.md

```diff
@@ -756,15 +767,15 @@ Main circuit could be splitted in the following phases:
 |        ethAddr2[nTx]         |   uint160 array   |                      ethAddr of the receiver leaf                       |
 | siblings2[nTx][nLevels + 1]  |    field array    |               siblings merkle proof of the receiver leaf                |
 |        isOld0_2[nTx]         |    bool array     |                     flag to require old key - value                     |
-|         oldKey2[nTx]         |   uint48 array    |                       old key of the sender leaf                        |
+|         oldKey2[nTx]         |   uint40 array    |                       old key of the sender leaf                        |
 |        oldValue2[nTx]        |    field array    |                      old value of the sender leaf                       |
 |      tokenID3[maxFeeTx]      |   uint32 array    |                        tokenID of leafs feeIdxs                         |
 |       nonce3[maxFeeTx]       |   uint40 array    |                         nonce of leafs feeIdxs                          |
 |       sign3[maxFeeTx]        |    bool array     |                          sign of leafs feeIdxs                          |
 |      balance3[maxFeeTx]      |   uint192 array   |                        balance of leafs feeIdxs                         |
 |        ay3[maxFeeTx]         |    field array    |                           ay of leafs feeIdxs                           |
 |      ethAddr3[maxFeeTx]      |   uint160 array   |                        ethAddr of leafs feeIdxs                         |
-|    siblings3[nLevels + 1]    | field array       |                   siblings merkle proof of leafs Idxs                   |
+|    siblings3[nLevels + 1]    | field array array |                   siblings merkle proof of leafs Idxs                   |
```

Comment:
Missing `[maxFeeIdxs]`. Apart, at the 2nd column appears the `field array array`, while at the same column for the row of `siblings1` and `siblings2` is also an *array of arrays of field elements* but only appears as `field array` (lines 758 and 768) (would affect also parameter descriptions such as the `FromBJJCompressed`)
```suggestion
|    siblings3[maxFeeIdxs][nLevels + 1]    | field array array |                   siblings merkle proof of leafs Idxs                   |
```

Reply:
Added `array array` in all inputs consisting in *array of arrays of field elements*